wfe/wfe.go

```diff
@@ -1391,7 +1347,6 @@ func (wfe *WebFrontEndImpl) Certificate(ctx context.Context, logEvent *requestEv
 	// Certificate paths consist of the CertBase path, plus exactly sixteen hex
 	// digits.
 	if !core.ValidSerial(serial) {
-		logEvent.AddError("certificate serial provided was not valid: %s", serial)
```

Comment:
This seems like a case where we now won't be able to identify that a given "Certificate not found" was due to an invalid serial, or what the serial was. Maybe not important but I thought it should be flagged.

Reply:
This is another situation where we're doing basic input validation, e.g. the user queried "/acme/cert/poodle", and we don't need to log more detail; it's self evident from the logged URL.